
<p align="center">
<h1 align="center"> <img src="[https://github.com/xcfcode/Summarization-Papers/pic/summary.png](https://github.com/xcfcode/Summarization-Papers/blob/main/pic/summary.png)" width="30" />Papers of Machine Learning for Genomics</h1>
</p>
Papers on machine learning for proteins


## Contributor
Organized by [Robert Tang](https://xiangrutang.github.io/).

## Survey
- **Genomics enters the deep learning era** *Etienne Routhier, Julien Mozziconacci*
- **Machine Learning Methods for Exploring Sequence Determinants of 3D Genome Organization** *Muyu Yang, Jian Ma*
- **Computational and experimental methods for classifying variants of unknown clinical significance** *Malte Spielmann, Martin Kircher*
- **Fine-Tuning Transformers For Genomic Tasks** *Vlastimil Martinek, David Cechak, Katarina Gresova, Panagiotis Alexiou, Petr Simecek* [[paper]](https://www.biorxiv.org/content/10.1101/2022.02.07.479412v1.abstract)



## Transformers
- **DNABERT: pre-trained Bidirectional Encoder Representations from Transformers
model for DNA-language in genome** *Yanrong Ji, Zhihan Zhou, Han Liu, Ramana V Davuluri* `Bioinformatics 2021` [[paper]](https://academic.oup.com/bioinformatics/article/37/15/2112/6128680)

- **Effective gene expression prediction from sequence by integrating long-range interactions** *Žiga Avsec, Vikram Agarwal, Daniel Visentin, Joseph R. Ledsam, Agnieszka Grabska-Barwinska, Kyle R. Taylor, Yannis Assael, John Jumper, Pushmeet Kohli & David R. Kelley* `Nature Methods 2021` [[paper]](https://www.nature.com/articles/s41592-021-01252-x)

- **BERT-Promoter: An improved sequence-based predictor of DNA promoter using BERT pre-trained model and SHAP feature selection** *Nguyen Quoc Khanh Le, Quang-Thai Ho, Van-Nui Nguyen, Jung-Su Chang* [[paper]](https://www.sciencedirect.com/science/article/pii/S1476927122001128?casa_token=KrR7rtlA4DoAAAAA:pmWZevNjaDzCs_jVsHQgU_-FibyfD6jIAZJlZRlEya7jEAI0wfAzLZSvq2sGIdz-p8Q42YALgw)

- **SNP2Vec: Scalable Self-Supervised Pre-Training for Genome-Wide Association Study** *Samuel Cahyawijaya, Tiezheng Yu, Zihan Liu, Tiffany T.W. Mak, Xiaopu Zhou, Nancy Y. Ip, Pascale Fung* [[paper]](https://arxiv.org/abs/2204.06699)

- **Transformer-based embedding applied to classify bacterial species using sequencing reads** *Ho-Jin Gwak; Mina Rho* 

## Functional Interactions
- **Co-evolution based machine-learning for predicting functional interactions between human genes** *Doron Stupp, Elad Sharon, Idit Bloch, Marinka Zitnik, Or Zuk, Yuval Tabach* `Nature Communications 2021` [[paper]](https://www.nature.com/articles/s41467-021-26792-w)

## Gene Expression & Regulation
- **Integrating long-range regulatory interactions to predict gene expression using graph convolutional networks** *Jeremy Bigness, Xavier Loinaz, Shalin Patel, Erica Larschan, Ritambhara Singh*
- **Assessing comparative importance of DNA sequence and epigenetic modifications on gene expression using a deep convolutional neural network**
- **Predicting Tissue-Specific mRNA and Protein Abundance in Maize: A Machine Learning Approach** *Kyoung Tak Cho, Taner Z. Sen, Carson M. Andorf* `Frontiers in Artificial Intelligence` [[paper]](https://static.frontiersin.org/articles/10.3389/frai.2022.830170/full)
- **The evolution, evolvability and engineering of gene regulatory DNA** *Eeshit Dhaval Vaishnav, Carl G. de Boer, Jennifer Molinet, Moran Yassour, Lin Fan, Xian Adiconis, Dawn A. Thompson, Joshua Z. Levin, Francisco A. Cubillos & Aviv Regev* `Nature 2022`
- **Chromatin interaction–aware gene regulatory modeling with graph attention networks** *Alireza Karbalayghareh, Merve Sahin and Christina S. Leslie*
- **Transcriptional enhancers at 40: evolution of a viral DNA element to nuclear architectural structures**

## Transcription Factors
- **DeepSTARR predicts enhancer activity from DNA sequence and enables the de novo design of synthetic enhancers** *Bernardo P. de Almeida, Franziska Reiter, Michaela Pagani & Alexander Stark* `Nature Genetics 2022`
- **Generating specificity in genome regulation through transcription factor sensitivity to chromatin** *Luke Isbel, Ralph S. Grand & Dirk Schübeler* `Nature Reviews Genetics (2022)`

## Genetic Perturbation
- **GEARS: Predicting transcriptional outcomes of novel multi-gene perturbations** *Yusuf Roohani, Kexin Huang, Jure Leskovec* 

## Interpretability
- **Assessment and Optimization of the Interpretability of Machine Learning Models Applied to Transcriptomic Data** *Yongbing Zhao, Jinfeng Shao, Yan W Asmann*
- **ExplaiNN: interpretable and transparent neural networks for genomics** *Gherman Novakovsky, Oriol Fornes, Manu Saraswat, Sara Mostafavi, Wyeth W. Wasserman**

## Others
- **GenomeWarp: an alignment-based variant coordinate transformation** *Cory Y. McLean, Yeongwoo Hwang, Ryan Poplin, Mark A. DePristo* `Bioinformatics (2019)` [[paper]](https://academic.oup.com/bioinformatics/article/35/21/4389/5420550)
- **Predicting the sequence specificities of DNA-binding proteins by DNA Fine-tuned Language Model with decaying learning rates** *Ying He; De-shuang Huang; Qinhu Zhang; Siguo Wang; Zhan-Heng Chen; Zhen Cui; Zhen-Hao Guo* [[paper]](https://ieeexplore.ieee.org/abstract/document/9751352/authors#authors)

- **CONCERT: Genome-wide prediction of sequence elements that modulate DNA replication timing** *Yang Yang, Yuchuan Wang, Yang Zhang, Jian Ma1*

## Tools
- **Ensembl 2022**
